tests/template_tests/syntax_tests/test_if.py

```diff
@@ -601,6 +601,12 @@ def test_if_is_not_both_variables_missing(self):
         output = self.engine.render_to_string('template', {})
         self.assertEqual(output, 'no')
 
+    @setup({'if-tag-stream': '{% if foo %}{{ a }}42{% else %}no{% endif %}'})
+    def test_if_tag_stream(self):
+        output = self.engine.render_to_string('if-tag-stream', {'foo': True, 'a': 21}, stream=True)
+        self.assertEqual(next(output), '21')
+        self.assertEqual(next(output), '42')
```

Comment:
Please remove this and other tests related with adding `stream` argument to the `render_to_string()` and revert also docs changes.